### DISCUSSION TITLE
网关遇到一个问题

### DISCUSSION THREAD

**Discussion by Leno709394** [kkkgo/PaoPaoGateWay#14] [问和答]
我的是多节点的，一旦目前连的节点不行，无论是等这个节点后面自行恢复正常了。还是自动切换或手动切换了别的节点了。都访问不了外网了。必须重启网关才可以。 已经更新到最新版本了

**Comment by Leno709394** [comment]
泡泡这套东西是真的好用。现在就剩这个问题很烦人

**Comment by kkkgo** [comment]
按道理fast_node自动切换是会切换到可用节点。可以大概描述一下你使用的配置吗，还有不能访问的时候，虚拟机有什么滚动日志输出。可以使用gifcam录制gif。

**Reply by Leno709394** [reply]
i5-12400  技嘉B760m   16Gx2     
win2022 server hyperv   爱快主路由 静态指向paopaogateway  dns设为paopao dns，
刚重启过。现在正常。等出问题。我截下日记

**Comment by kkkgo** [comment]
现在确认了一个问题，7月份新引入的功能导致了bug。稍后修复。

**Comment by kkkgo** [comment]
使用最新release试试。https://github.com/kkkgo/PaoPaoGateWay/releases

**Reply by Leno709394** [reply]
辛苦了。早上起来又断了，今天更新测试下。
